src/AsmResolver.DotNet/Serialized/SerializedGenericParameter.cs

```diff
@@ -31,7 +31,7 @@ public SerializedGenericParameter(ModuleReaderContext context, MetadataToken tok
             _row = row;
 
             Attributes = _row.Attributes;
-            Number = row.Number;            
+            //Number = row.Number;
```

Comment:
Remove comment

Reply:
You can't set the number manually now. Setting the number would be done through reordering the owner array.